src/pages/home/sidebar/OptionRow.js

```diff
@@ -275,6 +288,14 @@ export default withLocalize(memo(OptionRow, (prevProps, nextProps) => {
         return false;
     }
 
+    if (!_.isEqual(prevProps.textStyles, nextProps.textStyles)) {
+        return false;
+    }
+
+    if (!_.isEqual(prevProps.alternateTextStyles, nextProps.alternateTextStyles)) {
```

Comment:
can be combined into the previous if statement

Reply:
okay